shared/test/test_regexp.rb

```diff
@@ -0,0 +1,53 @@
+require_relative 'test_helper'
+
+require 'cdo/regexp'
+class RegexpUtilsTest < Minitest::Test
+  EXPECTED_DIGITS = '1234567890'
+  VALID_US_PHONE_NUMBERS = %w[
```

Comment:
@tanyaparker @aoby hold on, keep in mind this is used for avoiding letting users share programs with PII. That would mean programs with strings of digits would count as containing phone numbers, and would then not share. It sounds like we might want two different utilities in this case.